Makefile

```diff
@@ -1,6 +1,6 @@
 PWD = $(shell pwd)
 
-all: lint build
+all: lint build preview
```

Comment:
Can we revert this please?

Reply:
I'll revert but I think it makes sense to also generate the preview. For me at least, I test it locally so I always have to run make twice.

 :ok: by @regularjack on 24/04/2018 11:13:51